fluidBuild.config.cjs

```diff
@@ -206,6 +205,7 @@ module.exports = {
 				"tools/markdown-magic/package.json",
 			],
 			"npm-strange-package-name": [
+				"server/routerlicious/packages/package.json",
```

Comment:
I'd like to double check we actually need this. t9s is already listed under the known unscoped packages section, so ideally I'd expect we don't need to add this exception? But not sure.

Reply:
You're correct. `tinylicious` can be removed from the field now.